src/controllers/TracksController.php

```diff
@@ -22,4 +22,79 @@ public function getAction($request, $db)
 
         return $list;
     }
+
+    public function editTrack($request, $db)
+    {
```

Comment:
Can this be called without being authorized? I'm missing the authorization check...